### PR TITLE
Use updated import of testRepoHeaderForAll, and unskip tablesNoDocs

### PR DESCRIPTION
**Pull request by katiemcculloch**

### CODE REVIEW

File: cypress/integration/utils/sharedTests/sqlEditor.ts [hunk 1]
```diff
@@ -8,20 +8,20 @@ import {
 const shouldArgs = newShouldArgs("be.visible");
 
 const sqlEditorClickFlow = newClickFlow(
-  "[data-cy=launch-sql-editor-button]",
+  "[data-cy=sql-editor-collapsed]",
```
Comment (katiemcculloch) [inline cypress/integration/utils/sharedTests/sqlEditor.ts:11]: Just noticed i have to update the descriptions in this file

File: cypress/integration/utils/sharedTests/repoHeaderNav.ts [hunk 2]
```diff
@@ -168,7 +168,7 @@ export const testRepoHeaderForAll = (
 export const testRepoHeaderWithBranch = (
   repoName: string,
   ownerName: string,
-  loggedIn = false,
```
Comment (tbantle22) [inline cypress/integration/utils/sharedTests/repoHeaderNav.ts:171]: Can you leave this so you don't have to pass loggedIn to this function for every page?

**Review by tbantle22 — APPROVED** [review]
One comment, otherwise looks good

**Comment by katiemcculloch** [comment]
@tbantle22 thx for review! i'm going to push feedback in another PR so i can get these tests on master now

**Pull request merged**